tests/sles4sap/crash/azure/crash_deploy.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: This module is responsible for creating all necessary Azure resources:
+# - Azure Resource Group
+# - Network Security Group and SSH rule
+# - Virtual Network and Subnet
+# - Public IP and NIC
+# - VM creation from image or VHD blob
+# It saves VM public IP and SSH command into job variables
+
+use base 'publiccloud::basetest';
+use testapi;
+use publiccloud::utils;
+use mmapi 'get_current_job_id';
+use serial_terminal 'select_serial_terminal';
+use sles4sap::azure_cli;
+use utils;
+use version_utils;
+
+use constant DEPLOY_PREFIX => 'clne';
```

Comment:
I think it's not a good idea to have the deployment prefix defined as a constant across various modules. It means that if we ever need to change it, we have to do it in more than one place. I propose instead to move this to an openQA setting, which we can get with `get_required_var('DEPLOY_PREFIX')` making it mandatory, or that we can get with `get_var('DEPLOY_PREFIX, 'clne')` in the first module and with `get_required_var('DEPLOY_PREFIX')` everywhere else so that the default value is a constant defined only in one place.